app/models/work.rb

```diff
@@ -34,19 +34,20 @@ class Work < ApplicationRecord
     end
 
     event :begin_deposit do
-      transition first_draft: :depositing, version_draft: :depositing, pending_approval: :depositing
+      transition %i[first_draft version_draft pending_approval] => :depositing
     end
 
     event :deposit_complete do
       transition depositing: :deposited
     end
 
     event :submit_for_review do
-      transition first_draft: :pending_approval
+      transition %i[first_draft version_draft] => :pending_approval
     end
 
     event :reject do
-      transition pending_approval: :first_draft
+      transition pending_approval: :first_draft, if: ->(work) { work.druid.blank? }
```

Comment:
This is the next bug we'd hit: having a version_draft be submitted for review and then returning to first_draft.

Reply:
Do you think this should go to a returned state? See https://github.com/sul-dlss/happy-heron/issues/576

Reply:
I'll take a look.

Reply:
@jcoyne can I defer this change to #576, and pick that up as my next ticket?

Reply:
Sure